### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by ldragoon**
Decoding obj.loc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
     "path": "^0.12.7",
     "traverse": "^0.6.6",
     "yargs": "^14.2.0"
+  },
+  "devDependencies": {
+    "fetch": "^1.1.0"
```
Comment (qjhart) [inline package.json]: @ldragoon  Do we need this dependancy anymore?

File: Makefile [hunk 2]
```diff
@@ -2,13 +2,17 @@
 SHELL:=/bin/bash
 
 # Need Riot installed
-riot:=~/apache-jena-3.8.0/bin/riot
+riot:=/usr/local/Cellar/jena/3.13.1/bin/riot
```
Comment (qjhart) [inline Makefile]: @ldragoon I guess we should leave this as 'riot' and force us to add to our path

File: Makefile [hunk 3]
```diff
@@ -2,13 +2,17 @@
...
 
 key:=supply_on_cmd
 token:=supply_token_on_cmd
 
-board:=NgVOlKPZ
-moment:=chardonney
+#board:=NgVOlKPZ
+#moment:=chardonney
+
+board:=K2CNYMZP
```
Comment (qjhart) [inline Makefile]: @ldragoon This will be where our example will go as well.

File: location.txt [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
```
Comment (qjhart) [inline location.txt]: @ldragoon  We don't need this file in master

Comment (ldragoon) [reply location.txt]: Removed this file.

File: package.json (hunk 1 above)
Comment (ldragoon) [reply package.json]: Looks like it's no longer in use, I removed it.

File: Makefile [hunk 5]
```diff
@@ -36,5 +41,8 @@ thumbnails: ${board}/board.json
 
 ${moment}.ttl:${board}/board.json
 	./trello2moment --board=${board} 2>${moment}.err > ${moment}_t.ttl
-	${riot} --formatted=ttl --base=z: ${moment}_t.ttl  | sed -e 's/<z:/</g' > $@
+	riot --formatted=ttl --base=z: ${moment}_t.ttl  | sed -e 's/<z:/</g' > $@
```
Comment (qjhart) [inline Makefile]: @ldragoon I hate to keep on this, but now you are not using the variable you described above.

**Pull request merged**